src/Expander.hs

```diff
@@ -1166,19 +1166,19 @@ requireDeclarationCat :: Syntax -> MacroDest -> Expand (DeclTreePtr, DeclOutputS
 requireDeclarationCat _ (DeclTreeDest dest outScopesDest) = return (dest, outScopesDest)
 requireDeclarationCat stx other =
   throwError $
-  WrongSyntacticCategory stx (actual DeclarationCat) (expected $ problemCategory other)
+  WrongSyntacticCategory stx (mortise DeclarationCat) (tenon $ problemCategory other)
```

Comment:
This is again backwards... `requireDeclarationCat` checks if the _destination_ expects a declaration, so `other` is the `mortise`, not the `tenon`.

Reply:
:face_exhaling: good thing the type system will eventually encode this because I keep messing it up!